public/app/features/alerting/unified/components/mute-timings/useMuteTimings.tsx

```diff
@@ -121,43 +124,29 @@ export const useMuteTimings = ({ alertmanager }: BaseAlertmanagerArgs) => {
  *
  * Otherwise, creates the new timing in `time_intervals` via AM config API
  */
+type CreateMuteTimingArgs = { interval: MuteTimeInterval };
```

Comment:
(I've pushed a commit to just move this above the jsdoc comment so that comment is still attributed to the `useCreateMuteTiming` hook instead of this type. I've also just re-used this in a couple of additional places - WDYT?)

Reply:
Oops, definitely was supposed to be linked to the function and not the type :D